include/penguin.h

```diff
@@ -13,19 +13,22 @@ extern "C" {
 
 // attenuation coefficient TODO: validate values
```

Comment:
Maybe we should open an issue for those as well. It's easier to keep track of issues than TODOs in the code. Plus then we can discuss our thoughts in the comments.